rosplan_knowledge_msgs/msg/DomainFormula.msg

```diff
@@ -1,4 +1,5 @@
 # A knowledge item used to represent an atomic formula from the domain.
 # typed_parameters matches label -> type
 string name
+string[] labels
```

Comment:
I'm not convinced this is required. The DomainFormula message is used to store the ungrounded predicate, mapping label->type.

I have also used it to store a grounded action as an intermediate step towards KnowlegeItem, in which case I mapped label->object. (KnowledgeItem does not store the types of its parameters).

If labels has been added, then what is being stored in Key?